lightning/src/util/config.rs

```diff
@@ -48,6 +48,30 @@ pub struct ChannelHandshakeConfig {
 	/// Default value: 1. If the value is less than 1, it is ignored and set to 1, as is required
 	/// by the protocol.
 	pub our_htlc_minimum_msat: u64,
+	/// Sets the percentage of the channel value we will cap the total value of outstanding inbound
+	/// HTLCs to.
+	///
+	/// This can be set to a value between 1-100, where the value corresponds to the percent of the
+	/// channel value in whole percentages.
+	///
+	/// Note that:
+	/// * If configured to another value than the default value 10, any new channels
+	/// created with the non default value will cause versions of LDK prior to 0.0.100 to refuse to
```

Comment:
Hmm could you please check the part about "LDK prior to 0.0.100" here? The `holder_max_htlc_value_in_flight_msat` was added to the TLV fields in LDK 0.0.104 ( f69311c ). 

As I'm still a bit uncertain about our serialization:
As the field was added with an is even number (`6`), does that mean that versions prior to LDK 0.0.103 will also fail when deserializing the `Channel` (hence this should be changed)?

Reply:
Thanks for clarifying :)